subtitles.go

```diff
@@ -176,14 +176,14 @@ type StyleAttributes struct {
 	SSAMarginRight       *int // pixels
 	SSAMarginVertical    *int // pixels
 	SSAMarked            *bool
-	SSAOutline           *int // pixels
+	SSAOutline           *float64 // pixels
```

Comment:
Those changes are fine but they break tests (`go test ./...`). You need to do 2 things:

- update `ssa_test.go` so that those attributes are `astiptr.Float` instead of `astiptr.Int`
- update `testdata/example-out.ssa` so that some values in the `Style` section are with `.000` at the end